rsv-consumers/libexec/consumers/html-consumer

```diff
@@ -114,6 +114,12 @@ class HTMLConsumer(RSVConsumer.RSVConsumer):
         except EOFError, err:
             # Should this be a warning?
             self.log("State file is empty")
+        except ValueError as err:
```

Comment:
As long as we're catching `ValueError`, we might as well be catching [some of the other exceptions that come up during unpickling](https://docs.python.org/2.6/library/pickle.html#pickle.UnpicklingError) - say `AttributeError` and `IndexError`. (They don't have to be separate blocks, you could just do
`except (ValueError, AttributeError, IndexError) as err`)